src/Equinor.Procosys.Library.Query/GetFunctionalRoles/PersonsInFunctionalRole.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Equinor.Procosys.Library.Query.GetFunctionalRoles
+{
+    public class PersonsInFunctionalRole 
```

Comment:
PersonInFunctionalRole instead of PersonsInFunctionalRole? And could we get rid of the Persons object? Or is this not necessary?

Reply:
There is a PersonInFunctionalRole and a PersonsInFunctionalRole. I tried a few differet ways of doing this and it was the only way I was able to acquire any data.